keps/sig-node/24-apparmor/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: Add AppArmor Support
+kep-number: 24
+authors:
+  - "@saschagrunert"
+  - "@tallclair"
```

Comment:
I think you can remove me from the authors list. I don't think I should be both an author and reviewer :)